nexus/src/app/sagas/instance_create.rs

```diff
@@ -237,106 +339,108 @@ async fn sic_alloc_server(
+    let interface_id = repeat_saga_params.new_id;
+    let (.., authz_instance) = LookupPath::new(&opctx, &datastore)
+        .instance_id(instance_id)
+        .lookup_for(authz::Action::Modify)
+        .await
+        .map_err(ActionError::action_failed)?;
+    match LookupPath::new(&opctx, &datastore)
+        .network_interface_id(interface_id)
+        .lookup_for(authz::Action::Delete)
+        .await
+    {
+        Ok((.., authz_interface)) => {
+            datastore
+                .instance_delete_network_interface(
+                    &opctx,
+                    &authz_instance,
+                    &authz_interface,
+                )
                 .await
-            {
-                Ok((.., authz_interface)) => {
-                    datastore
-                        .instance_delete_network_interface(
-                            &opctx,
-                            &authz_instance,
-                            &authz_interface,
-                        )
-                        .await
-                        .map_err(|e| e.into_external())?;
-                    Ok(())
-                }
-                Err(Error::ObjectNotFound { .. }) => {
-                    // The saga is attempting to delete the NIC by the ID cached
-                    // in the saga log. If we're running this, the NIC already
-                    // appears to be gone, which is odd, but not exactly an
-                    // error. Swallowing the error allows the saga to continue,
-                    // but this is another place we might want to consider
-                    // bumping a counter or otherwise tracking things.
-                    warn!(
-                        osagactx.log(),
-                        "During saga unwind, NIC already appears deleted";
-                        "interface_id" => %id,
-                    );
-                    Ok(())
-                }
-                Err(e) => Err(e.into()),
-            }
+                .map_err(|e| e.into_external())?;
+            Ok(())
         }
+        Err(Error::ObjectNotFound { .. }) => {
+            // The saga is attempting to delete the NIC by the ID cached
+            // in the saga log. If we're running this, the NIC already
+            // appears to be gone, which is odd, but not exactly an
+            // error. Swallowing the error allows the saga to continue,
+            // but this is another place we might want to consider
+            // bumping a counter or otherwise tracking things.
+            warn!(
+                osagactx.log(),
+                "During saga unwind, NIC already appears deleted";
+                "interface_id" => %interface_id,
+            );
+            Ok(())
+        }
+        Err(e) => Err(e.into()),
     }
 }
 
-/// Create one custom (non-default) network interfaces for the provided instance.
+/// Create one custom (non-default) network interfaces for the provided
```

Comment:
My original error, but spelling: `interfaces` -> `interface`.

Reply:
Fixed in 96dba6366f6cb8b47ff10a3a4b0bf0277f9d060e.